demo/setup.py

```diff
@@ -32,7 +32,7 @@ def read_relative_file(filename):
     keywords='class-based view, generic view, download',
     author=u'Rémy HUBSCHER',
```

Comment:
You might want to change that too I guess, PeopleDoc Inc. is the actual owner of the copyright.

Reply:
it can stay like this. author != owner :)